pineappl/src/grid.rs

```diff
@@ -1446,16 +1466,21 @@ impl Grid {
     pub fn evolve_with_slice_iter<'a, E: Into<anyhow::Error>>(
```

Comment:
I'm uncertain whether always passing two EKOs is a good idea. Note that this is a performance-critical section part of PineAPPL, and we don't want to unnecessarily slow down FK-table generation. Some jet grids take up to five hours if I remember correctly. We must check the impact of this patch on the performance.

Reply:
@giacomomagni said that the longest jet now takes about 30mn. But I absolutely agree that we should thoroughly test how much does this affect the performance, and if it does in some significant way implement something better.

Reply:
This shouldn't have an impact on the runtime since we have two separate operators in any case. If we pass the same operator twice we may waste some memory, but that's a problem we can address later.